app/src/main/res/layout/fragment_drawing.xml

```diff
@@ -55,6 +55,21 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/drawSpace" />
 
+    <Button
+        android:id="@+id/save_button"
```

Comment:
maybe you could precise this id.
for example: `"@+id/darwing_fragment_save_button"`